{"cveId": "CVE-2021-47518", "cwe": ["CWE-476", "CWE-476"], "cvss": [{"baseScore": "5.5", "version": "3.1", "Attack Vector": "Low", "Attack Complexity": "Low", "Privileges Required": "Low", "User Interaction": "None", "Scope": "Unchaged", "Confidentiality Impact": "None", "Integrity Impact": "None", "Availability Impact": "High"}], "references": ["https://git.kernel.org/stable/c/87cdb8789c38e44ae5454aafe277997c950d00ed", "https://git.kernel.org/stable/c/69bb79a8f5bb9f436b6f1434ca9742591b7bbe18", "https://git.kernel.org/stable/c/811a7576747760bcaf60502f096d1e6e91d566fa", "https://git.kernel.org/stable/c/3b861a40325eac9c4c13b6c53874ad90617e944d", "https://git.kernel.org/stable/c/48fcd08fdbe05e35b650a252ec2a2d96057a1c7a", "https://git.kernel.org/stable/c/83ea620a1be840bf05089a5061fb8323ca42f38c", "https://git.kernel.org/stable/c/fae9705d281091254d4a81fa2da9d22346097dca", "https://git.kernel.org/stable/c/4cd8371a234d051f9c9557fcbb1f8c523b1c0d10"], "description": ["In the Linux kernel, the following vulnerability has been resolved:\n\nnfc: fix potential NULL pointer deref in nfc_genl_dump_ses_done\n\nThe done() netlink callback nfc_genl_dump_ses_done() should check if\nreceived argument is non-NULL, because its allocation could fail earlier\nin dumpit() (nfc_genl_dump_ses())."], "published": "2024-05-24T15:15:13.557", "state": "PUBLIC", "vendorName": ["Linux"], "productName": ["Linux"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}